### PR TITLE
apply --paper-font-common-base

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Applies `--paper-font-common-base` to the host.

This is part of the effort of applying `Roboto` font to all the paper elements http://jsbin.com/mumovix/1/edit?html,output

### CODE REVIEW

**Comment by valdrinkoshi** [comment]
@cdata @notwaldorf @bicknellr kindly review

File: paper-tab.html [hunk 1]
```diff
@@ -36,6 +37,7 @@
 `--paper-tab` | Mixin applied to the tab | `{}`
 `--paper-tab-content` | Mixin applied to the tab content | `{}`
 `--paper-tab-content-unselected` | Mixin applied to the tab content when the tab is not selected | `{}`
+`--paper-font-common-base` | Font mixin | Defined in `paper-styles/typography.html`
```
Comment (bicknellr) [inline paper-tab.html]: I don't think we should advertise this mixin since it's not intended to be overwritten; `--paper-tab` should cover that case.

**Comment by bicknellr** [comment]
LGTM after comments.

File: paper-tab.html (hunk 1 above)
Comment (valdrinkoshi) [inline paper-tab.html]: I don't have strong opinion on this one.
@notwaldorf @cdata WDYT? I'll update all PRs accordingly

Comment (abdonrd) [inline paper-tab.html]: I'm agree with @bicknellr!

Comment (notwaldorf) [inline paper-tab.html]: Yeah, I think what goes in that docs section is "mixins this element has defined", not "mixins this element has used". For example, we don't mention all the flexbox ` --layout-inline` and friends mixins we're using either.

Comment (valdrinkoshi) [inline paper-tab.html]: agreed, update coming!

**Pull request merged**